introduction.md

```diff
@@ -1,2 +1,7 @@
 I am Falk, Johan Falk.I
-My favorite codeschool path is the JavaScript path.
\ No newline at end of file
+My favorite codeschool path is the JavaScript path.
+
+Deadly skills
+* Git
+* JavaScript
+* Ruby
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?